label_studio/projects/functions/utils.py

```diff
@@ -26,3 +26,17 @@ def make_queryset_from_iterable(tasks_list):
                 raise ValueError(f'Unknown object type: {str(task)}')
         queryset = Task.objects.filter(id__in=ids)
     return queryset
+
+
+def recalculate_created_labels_from_scratch(project, summary):
+    summary.created_labels = {}
```

Comment:
should we also clear out `summary.created_annotations` here? If so, we should probably rename the function. If not, I'm concerned that `update_created_annotations_and_labels` may further mess up `summary.created_annotations`

Reply:
you are right, added created_annotations too and renamed the func.